src/services/logging.service.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { Logger } from "./logging.service";
+import { mock, mockReset } from "jest-mock-extended";
+
+describe("LoggingService Unit Tests", () => {
+  let logger: Logger;
+  let internal: Console;
+
+  beforeAll(() => {
+    internal = mock<Console>();
+    logger = new Logger(internal);
+  });
+
+  afterEach(() => {
+    Logger.debug = true;
+    // Reset the mock after the JS stack has cleared
+    mockReset(internal);
+  });
+
+  it("should be defined", () => {
+    expect(Logger).toBeDefined();
+  });
```

Comment:
We can remove this test as it's also covered by TypeScript. I guess generally speaking this matcher is rarely needed in a TypeScript project (if at all?). The same applies for testing methods with incorrect parameter types (although tests are written in TypeScript so they won't actually compile 😆)

Reply:
I generally do that one because I create the file at the same time as the code and jest crashes if there is an empty spec file. 👍